popUp.css

```diff
@@ -0,0 +1,194 @@
+#overlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  width: 100vw;
+  height: 108vh;
+  opacity: 0.9;
+  z-index: 100;
+  display: none;
+}
+
+.popUp {
+  width: 90vw;
+  margin-left: 20px;
+  padding: 8px 0 0 0;
+  font-family: "inter", sans-serif;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 5px;
+  background-color: white;
+  z-index: 200;
+}
```

Comment:
- Kindly make sure all parts in the popup are  displaying as expected I show in the picture below those buttons for `see live and see more` are not corrected displayed I can suggest you fix that can adjust a bit the size of the popup :+1: 


## your design 

![image](https://user-images.githubusercontent.com/95297251/204063245-0b12f1e5-fb19-4aae-a489-f3c6ad3bb598.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/204063492-58383e1d-3951-423b-88c4-46a736e5d233.png)